infrastructure/templates/public-api/deploy-stage-template.yml

```diff
@@ -81,13 +81,21 @@ stages:
               scriptLocation: inlineScript
               inlineScript: |
                 set -e
+                
                 az functionapp config appsettings set \
                   --name '$(dataProcessorFunctionAppName)' \
                   --resource-group '$(resourceGroupName)' \
                   --slot 'staging' \
                   --settings \
-                    "ConnectionStrings__PublicDataDb=@Microsoft.KeyVault(VaultName=$(keyVaultName); SecretName=$(dataProcessorPsqlConnectionStringSecretKey))" \
                     "CoreStorage=@Microsoft.KeyVault(VaultName=$(keyVaultName); SecretName=$(coreStorageConnectionStringSecretKey))"
+                
+                az webapp config connection-string set \
+                  --name '$(dataProcessorFunctionAppName)' \
+                  --resource-group '$(resourceGroupName)' \
+                  --slot 'staging' \
+                  --connection-string-type=PostgreSQL \
+                  --settings \
+                    "PublicDataDb=@Microsoft.KeyVault(VaultName=$(keyVaultName); SecretName=$(dataProcessorPsqlConnectionStringSecretKey))"
```

Comment:
Minor one, but `--connection-string-type 'PostgreSQL'` or `--connection-string-type PostgreSQL` looks like it should work and then be consistent with the other parameters.